### ISSUE TITLE
wimpsim-reader - default options are invalid (Trac #2155)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2147]
In [http://software.icecube.wisc.edu/documentation/inspect/wimpsim_reader.html?highlight=i3wimpsim#I3WimpSimReader I3WimpSimReader] we can read:

  Param EndMJD:	Default = nan, MJD to end simulation; if unspecified: read everything

But if I try not to set it (and take the NAN default) I receive this error

```text
ERROR (dataclasses): Calling with NAN not possible; will do nothing (I3Time.cxx:142 in void I3Time::SetModJulianTimeDouble(double))
```

The same is for `StartMJD`.

Migrated from https://code.icecube.wisc.edu/ticket/2155
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:23", 
    "description": "In [http://software.icecube.wisc.edu/documentation/inspect/wimpsim_reader.html?highlight=i3wimpsim#I3WimpSimReader I3WimpSimReader] we can read:\n\n  Param EndMJD:\tDefault = nan, MJD to end simulation; if unspecified: read everything\n\nBut if I try not to set it (and take the NAN default) I receive this error\n\n\n{{{\nERROR (dataclasses): Calling with NAN not possible; will do nothing (I3Time.cxx:142 in void I3Time::SetModJulianTimeDouble(double))\n}}}\n\nThe same is for `StartMJD`.\n\n", 
    "reporter": "grenzi", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067323910946", 
    "component": "combo simulation", 
    "summary": "wimpsim-reader - default options are invalid", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2018-05-17T15:48:52", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180517T15:49:36: nega changed description from:

> ----
> 
> 
> In [http://software.icecube.wisc.edu/documentation/inspect/wimpsim_reader.html?highlight=i3wimpsim#I3WimpSimReader I3WimpSimReader] we can read:
> 
>   Param EndMJD:	Default = nan, MJD to end simulation; if unspecified: read everything
> 
> But if I try not to set it (and take the NAN default) I receive this error
> 
> 
> {{{
> ERROR (dataclasses): Calling with NAN not possible; will do nothing (I3Time.cxx:142 in void I3Time::SetModJulianTimeDouble(double))
> }}}
> 
> The same is for `StartMJD`.
> 
> 

to:

> In [http://software.icecube.wisc.edu/documentation/inspect/wimpsim_reader.html?highlight=i3wimpsim#I3WimpSimReader I3WimpSimReader] we can read:
> 
>   Param EndMJD:	Default = nan, MJD to end simulation; if unspecified: read everything
> 
> But if I try not to set it (and take the NAN default) I receive this error
> 
> 
> {{{
> ERROR (dataclasses): Calling with NAN not possible; will do nothing (I3Time.cxx:142 in void I3Time::SetModJulianTimeDouble(double))
> }}}
> 
> The same is for `StartMJD`.
> 
>

**Comment by icecube-trac** [comment]
Trac update at 20180922T09:29:06: olivas changed milestone from "" to "Winter Solstice 2018"

**Comment by icecube-trac** [comment]
Trac update at 20181207T20:22:18: 
* mjl5147 commented:

> In changeset 167407:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="167407"
> Catch NAN start/end times, but warn the user during configuration. Closes #2147.
> ```


* mjl5147 changed status from "new" to "closed"
* mjl5147 changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T04:09:47: nwhitehorn commented:

> In changeset 170620:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170620"
> Catch NAN start/end times, but warn the user during configuration. Closes #2147.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:23: 
* olivas changed milestone from "Winter Solstice 2018" to ""
* olivas commented:

> Milestone Winter Solstice 2018 deleted
